### PR TITLE
Update cardano-cli to 8.18.0.0 and cardano-api to 8.37.0.0

### PR DESCRIPTION
**Pull request by smelc**
# Description

* Update to latest versions of cardano-api and cardano-cli

# Checklist

- [X] Commit sequence broadly makes sense and commits have useful messages
- NA New tests are added if needed and existing tests are updated.
- [X] Self-reviewed the diff

### CODE REVIEW

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/FoldBlocks.hs [hunk 1]
```diff
@@ -5,6 +5,7 @@ module Cardano.Testnet.Test.FoldBlocks where
 
 import           Cardano.Api hiding (cardanoEra)
 import qualified Cardano.Api as C
+import qualified Cardano.Api.Shelley as C
```
Comment (Jimbo4350) [inline cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/FoldBlocks.hs:8]: Can you qualify both of these as:
```
import qualified Cardano.Api as Api
import qualified Cardano.Api.Shelley as Api
```

File: cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/ProposeNewConstitution.hs [hunk 2]
```diff
@@ -408,9 +406,3 @@ filterRatificationState c (EpochBoundaryRatificationState (AnyRatificationState
   let constitutionAnchorHash = Ledger.anchorDataHash $ Ledger.constitutionAnchor (rState ^. Ledger.rsEnactStateL . Ledger.ensConstitutionL)
   in Text.pack c == renderSafeHashAsHex constitutionAnchorHash
 filterRatificationState _ _ = False
-
```
Comment (Jimbo4350) [inline cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/ProposeNewConstitution.hs:411]: :+1:

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

File: cardano-node-chairman/cardano-node-chairman.cabal [hunk 3]
```diff
@@ -73,7 +73,7 @@ test-suite chairman-tests
                       , cardano-crypto-class ^>= 2.1.2
                       , filepath
                       , hedgehog
-                      , hedgehog-extras ^>= 0.4.7.0
+                      , hedgehog-extras ^>= 0.5.0.0
```
Comment (newhoggy) [inline cardano-node-chairman/cardano-node-chairman.cabal:76]: The latest version is `hedgehog-extras-0.5.1.0`

**Review by carbolymer — APPROVED** [review]

**Comment by smelc** [comment]
Got beaten on the finish line by https://github.com/IntersectMBO/cardano-node/pull/5633

**Pull request closed without merging**